lumen/views/base.py

```diff
@@ -337,6 +336,12 @@ def from_spec(
             if is_ref(value):
                 refs[p] = value
                 value = state.resolve_reference(value)
+            if cls._is_param_function(p):
+                if isinstance(value, dict) and 'type' in value:
+                    func_spec = dict(value)
+                    module_ref = func_spec.pop('type')
+                    func = resolve_module_reference(module_ref)
+                    value = func.instance(**func_spec)
```

Comment:
Should there be a comment on what this does?

Reply:
Sure, if you can find something more descriptive to say than "deserialize".